### PR TITLE
Add frequently asked questions about AMP Ads

### PR DESCRIPTION
**Pull request by jasti**
A quick FAQ pointing to some links before posting the public blogpost for the fast fetch launch

### CODE REVIEW

File: ads/google/a4a/docs/a4a-readme.md [hunk 1]
```diff
@@ -53,3 +53,33 @@ If you are a creative agency, please express interest via [this form](https://go
 Please refer to the [Network Implementation Guide](./Network-Impl-Guide.md)
 Ad networks and ad servers can integrate with [Cloudflare](https://blog.cloudflare.com/firebolt/) who provide an AMP Ad verification services, enabling any independent ad provider to deliver faster, lighter, and more engaging ads.
 
+## Frequently Asked Questions
+
+#### Are there any AMP Ad samples?
+Yes. A number of great looking ads developed in AMP format can be found [here](https://ampbyexample.com/amp-ads/#amp-ads/experimental_ads). They use advanced components in AMP. They give the user a great experience while ensuring that the performance remains great. 
+
+#### Are there any tools to create AMP Ads?
+Yes. [Celtra](http://www.prnewswire.com/news-releases/celtra-partners-with-the-amp-project-showcases-amp-ad-creation-at-google-io-event-300459514.html) provides out of the box support for AMP ads in their ad creator platform. Other tools like Google Web Designer are also in the process of adding support.
```
Comment (keithwrightbos) [inline ads/google/a4a/docs/a4a-readme.md]: Add link to Google Web Designer?

File: ads/google/a4a/docs/a4a-readme.md [hunk 2]
```diff
@@ -53,3 +53,33 @@ If you are a creative agency, please express interest via [this form](https://go
...
+## Frequently Asked Questions
+
+#### Are there any AMP Ad samples?
+Yes. A number of great looking ads developed in AMP format can be found [here](https://ampbyexample.com/amp-ads/#amp-ads/experimental_ads). They use advanced components in AMP. They give the user a great experience while ensuring that the performance remains great. 
+
+#### Are there any tools to create AMP Ads?
+Yes. [Celtra](http://www.prnewswire.com/news-releases/celtra-partners-with-the-amp-project-showcases-amp-ad-creation-at-google-io-event-300459514.html) provides out of the box support for AMP ads in their ad creator platform. Other tools like Google Web Designer are also in the process of adding support. 
+
+#### How can I verify that an AMP Ad is valid?
+Depending on your development environment, there are a few options:
```
Comment (keithwrightbos) [inline ads/google/a4a/docs/a4a-readme.md]: Worth making more explicit that AMP creatives must be valid to get preferential render?

File: ads/google/a4a/docs/a4a-readme.md [hunk 3]
```diff
@@ -53,3 +53,33 @@ If you are a creative agency, please express interest via [this form](https://go
...
+Yes. [Celtra](http://www.prnewswire.com/news-releases/celtra-partners-with-the-amp-project-showcases-amp-ad-creation-at-google-io-event-300459514.html) provides out of the box support for AMP ads in their ad creator platform. Other tools like Google Web Designer are also in the process of adding support. 
+
+#### How can I verify that an AMP Ad is valid?
+Depending on your development environment, there are a few options: 
+- Use the [AMP validator NPM](https://www.npmjs.com/package/amphtml-validator) module to build your own
+- Use the [AMP validator](https://validator.ampproject.org/) for one off testing
+- Partner with [Cloudflare](https://blog.cloudflare.com/firebolt/) and use their public validator end point. 
+
+#### Do AMP Ads support 3rd party verification and validation? 
+Yes, there is native support for verification and validation using amp-analytics. (e.g. Google’s ActiveView integrates this way). There are also other vendors like MOAT that are actively implementing support for it.
```
Comment (keithwrightbos) [inline ads/google/a4a/docs/a4a-readme.md]: By validation you mean validation of viewability?  I fear you're mixing terms here making it unclear...

File: ads/google/a4a/docs/a4a-readme.md [hunk 4]
```diff
@@ -53,3 +53,33 @@ If you are a creative agency, please express interest via [this form](https://go
...
+- Partner with [Cloudflare](https://blog.cloudflare.com/firebolt/) and use their public validator end point. 
+
+#### Do AMP Ads support 3rd party verification and validation? 
+Yes, there is native support for verification and validation using amp-analytics. (e.g. Google’s ActiveView integrates this way). There are also other vendors like MOAT that are actively implementing support for it. 
+
+#### Does AMP Ads support timeline based animation? 
+Yes. Learn more about it [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-animation/amp-animation.md). 
+
+#### Does AMP Ads have an exit API? 
+Yes. Learn more about it [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-exit/amp-ad-exit.md).
```
Comment (keithwrightbos) [inline ads/google/a4a/docs/a4a-readme.md]: Is exit API a standard throughout the industry?  Perhaps expand on what you mean by exit?

File: ads/google/a4a/docs/a4a-readme.md (hunk 1 above)
Comment (jasti) [inline ads/google/a4a/docs/a4a-readme.md]: Done.

File: ads/google/a4a/docs/a4a-readme.md (hunk 2 above)
Comment (jasti) [inline ads/google/a4a/docs/a4a-readme.md]: Done

File: ads/google/a4a/docs/a4a-readme.md (hunk 3 above)
Comment (jasti) [inline ads/google/a4a/docs/a4a-readme.md]: Done

File: ads/google/a4a/docs/a4a-readme.md (hunk 4 above)
Comment (jasti) [inline ads/google/a4a/docs/a4a-readme.md]: Done

File: ads/google/a4a/docs/a4a-readme.md (hunk 1 above)
Comment (jasti) [inline ads/google/a4a/docs/a4a-readme.md]: @keithwrightbos Could you PTAL?

File: ads/google/a4a/docs/a4a-readme.md [hunk 5]
```diff
@@ -53,3 +53,35 @@ If you are a creative agency, please express interest via [this form](https://go
...
+Yes, there is native support for verification and viewability detection using amp-analytics. (e.g. Google’s ActiveView integrates this way). There are also other vendors like MOAT that are actively implementing support for it. 
+
+#### Does AMP Ads support timeline based animation? 
+Yes. Learn more about it [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-animation/amp-animation.md). 
+
+#### Most ads have tappable targets and configurable ad exits. Does AMP Ads have a similar mechanism? 
+Yes. Learn more about it [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-exit/amp-ad-exit.md). 
+
+#### Where can I learn more about AMP Ads?
+The public [website](http://ampproject.org/ads) is a good place to start.
```
Comment (lannka) [inline ads/google/a4a/docs/a4a-readme.md]: https?

Comment (jasti) [inline ads/google/a4a/docs/a4a-readme.md]: done

**Pull request merged**